### PR TITLE
Updated MonitorPage's GLClears so that third party handlers do not "blin...

### PR DESCRIPTION
**Pull request by grundyboy34**
I made a small change to MonitorPage to correct some issues that it was causing with GLClear which I've documented with comments.

### CODE REVIEW

File: RasterPropMonitor/Core/MonitorPage.cs [hunk 1]
```diff
@@ -576,11 +577,12 @@ public void RenderBackground(RenderTexture screen)
                     }
                     break;
                 case BackgroundType.Texture:
+                    //call clear before redraw of textures
                     GL.Clear(true, true, ourMonitor.emptyColorValue);
                     Graphics.DrawTexture(new Rect(0, 0, screen.width, screen.height), backgroundTexture);
                     break;
                 case BackgroundType.Handler:
-
+                    //No clear here as it would interfere with the handlers(Causing effects such as VesselViewer to blink)
```
Comment (MOARdV) [inline RasterPropMonitor/Core/MonitorPage.cs]: There needs to be an "else" block added to the inner if clause with a GL.Clear in it to handle the case where backgroundHandlerMethod returns false, but there's no signal texture, or noSignalTexture is false.  To see why, look at a ship that has no docking port and no target, and select the docking port cam view on the MFD.  In pseudo-code:

```
if(!backgroundHandlerMethod)
{
    if(texture && showNoSignal)
    {
        Graphics.DrawTexture
    }
    else
    {
        GL.Clear
    }
}
```

Comment (grundyboy34) [inline RasterPropMonitor/Core/MonitorPage.cs]: Ahh yes, that's the only case which I hadn't tested, if it ran out of power. Other than that, Plugins such as VesselViewer no longer blinked, solid texture screens were drawn correctly and blank "text" screens were as well.

Comment (MOARdV) [inline RasterPropMonitor/Core/MonitorPage.cs]: The problem is the GL.Clear here - VV assumes that RPM doesn't clear the screen every frame, which was true prior to the change I made.  So when it's not refreshing every frame, we get some ugliness.  Your change with the tweak I suggested does fix the problem without forcing VV to update, and I haven't seen any other problems yet (although I don't have Engine Ignitor, and I haven't had a chance to test with the new DPAI).

Comment (grundyboy34) [inline RasterPropMonitor/Core/MonitorPage.cs]: Ahh I see, thanks for explaining that little tweak. I would update mine but I don't have time today.

**Pull request merged**